### DISCUSSION TITLE
Conversão Implícita de string para float

### DISCUSSION THREAD

**Discussion by mplbruno** [balta-io/2801#62] [Perguntas e Respostas] [answered]
É curioso que se não colocar o tipo e só dizer que é uma variável var, por exemplo, ele recebe a informação do Console.ReadLine(); e executa a soma normalmente.

using System;
namespace Calculator
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.Clear();

            Console.WriteLine("Primeiro Valor:");
        
            var v1 = Console.ReadLine();

            Console.WriteLine("Segundo Valor:");
 
             var v2 = Console.ReadLine();

            Console.WriteLine("");
          
           var  resultado = v1 + v2;

            Console.WriteLine($"O resultado da soma é {resultado}"); //Os dois modos dão o mesmo resultado

        }
    }
}

**Comment by andrebaltieri** [comment, marked as answer]
Obrigado pelo feedback @mplbruno
